pkg/gadgets/advise/networkpolicy/advisor/testdata/connect-to-svc-duplicate.golden

```diff
@@ -28,4 +28,3 @@ spec:
   policyTypes:
   - Ingress
   - Egress
-status: {}
```

Comment:
should this be part of this PR?

Reply:
> What do you mean by this?

I wanted to understand why do we need this change. I thought maybe some linter removed it :laughing: 

> I just would like to understand why as no package bumped here could lead to this behavior.

From the [changelog](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.28.md) it seems they [removed](https://github.com/kubernetes/kubernetes/pull/115843) this field now in `v1.28.0`